### PR TITLE
UI Adjustments Home, Create Challenge, Profile, Onboarding

### PR DESCRIPTION
**Pull request by belze-bre**
# What was done? ✅
> Describe in a few words what have you done? 
> - TK1 Onboarding: text structure DEV
> - TK3 Onboarding: images related to what is being spoken DEV
> - TK7 Onboarding: change call to action button DEV
> - TK16 Change Home empty state: change to new animation (this one is small) DEV
> - TK17 Change Home empty state: rethink text sentence DEV
> - TK18 Profile Empty State change: add eyes animation DEV
> - TK22 Profile: add empty state animation DEV
> - TK24 Profile: add phrase to profile DEV
> - TK32 Create Challenge: card title spacing (between the lines) DEV
> - TK33 Create Challenge: change animation of successfully created challenge DEV
> - TK52 Change Home empty state: update phrase DEV

### CODE REVIEW

File: Fiero/Views/Login/Onboarding/OnboardingCard.swift [hunk 1]
```diff
@@ -13,33 +13,49 @@ struct OnboardingCard: View {
...
+        } else if onboardingId == 2 {
             ZStack {
                 VStack {
                     Image(image)
                         .resizable()
                         .aspectRatio(contentMode: .fit)
+                        .frame(height: UIScreen.main.bounds.height * 0.8, alignment: .center)
                         .frame(width: UIScreen.main.bounds.width * 0.8, alignment: .center)
-                        .padding(.horizontal, Tokens.Spacing.md.value)
+//                        .padding(.horizontal, Tokens.Spacing.md.value)
```
Comment (MarinaFX) [inline Fiero/Views/Login/Onboarding/OnboardingCard.swift:46]: aqui tbm

File: Fiero/Views/Login/Onboarding/OnboardingCard.swift [hunk 2]
```diff
@@ -13,33 +13,49 @@ struct OnboardingCard: View {
...
+        } else if onboardingId == 1 {
+            ZStack {
+                VStack {
+                    Image(image)
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                        .frame(height: UIScreen.main.bounds.height, alignment: .center)
+                        .frame(width: UIScreen.main.bounds.width, alignment: .center)
+                        .padding(.bottom, Tokens.Spacing.sm.value)
+//                        .padding(.horizontal, Tokens.Spacing.md.value)
```
Comment (MarinaFX) [inline Fiero/Views/Login/Onboarding/OnboardingCard.swift:58]: e aqui

**Review by MarinaFX — DISMISSED** [review]
MARAVILHOSA PERFECT BRENDA CASA COMIGO GORGEOUS XUXU TUDO TE AMO

File: Fiero/Views/Challenges/EmptyChallengesView.swift [hunk 3]
```diff
@@ -15,13 +15,15 @@ struct EmptyChallengesView: View {
 
         VStack {
             Spacer()
+            Spacer()
             
-            LottieView(fileName: "tonto", reverse: false, loop: true).frame(width: 350 , height: 200)
+            LottieView(fileName: "sad", reverse: false, loop: true).frame(width: 300 , height: 150)
```
Comment (illuminatyy) [inline Fiero/Views/Challenges/EmptyChallengesView.swift:20]: o ideal é não usar frame fixo, mas proporcional ao tamanho que ocupa na tela pra funcionar em todos os dispositivos

Comment (MarinaFX) [reply Fiero/Views/Challenges/EmptyChallengesView.swift:20]: Também concordo c a sugestão, vou deixar ela ativa ainda pelo feedback e para alterar no futuro, mas testei no iPhone SE e ficou ok, nao deu problema de layout entao por enquanto vou mergear p termos a dev atualizada

**Pull request merged**